vortex-array/src/serde.rs

```diff
@@ -40,16 +39,13 @@ impl dyn Array + '_ {
     /// little-endian u32 describing the length of the flatbuffer message.
     pub fn serialize(&self, ctx: &ArrayContext, options: &SerializeOptions) -> Vec<ByteBuffer> {
         // Collect all array buffers
-        let mut array_buffers = vec![];
```

Comment:
I admit changes to this file are a drive by when I was looking at stats serialisation